ingestion/functions/parsing/brazil_rio_de_janeiro/rio_de_janeiro.py

```diff
@@ -0,0 +1,149 @@
+import os
+import sys
+from datetime import datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            "common/python"))
+    import parsing_lib
+
+def convert_date(date_str: str,dataserver=True):
+    """ 
+    Convert raw date field into a value interpretable by the dataserver.
+
+    The date is listed in dd/mm/YYYY  format
+    """    
+    date = datetime.strptime(date_str.split(' ')[0], "%Y/%m/%d")
+    if not dataserver:
+        return date.strftime("%m/%d/%Y")
+    return date.strftime("%m/%d/%YZ")
+
+
+
+def convert_gender(raw_gender):
+    if raw_gender == "M":
+        return "Male"
+    if raw_gender == "F":
+        return "Female"
+    
+
+    
+def convert_location(raw_entry):
+    '''
+    Using residential municipality as only available location information.
+    '''
+    query_terms = [
+        term for term in [
+            raw_entry.get("municipio_res", ""),
+            "Rio de Janeiro, Brazil"]
+        if type(term) == str]
+    
+    return {"query":  ", ".join(query_terms)}
+    
+
+def convert_age(entry: float):
+    '''
+    Want to return a float specifying age in years. If age field is empty, return None.
+    There are some odd ages in this dataset (43 people over the age of 150). These erroneous ages
+    don't have a value for the FAIXA_ETARIA field, so will only return age if this field is populated
+    '''
+    if entry['idade']:
+        return float(entry['idade'])
+
+    
+            
+def parse_cases(raw_data_file, source_id, source_url):
+    """
+    Parses G.h-format case data from raw API data.
+    
+    Only include cases which include a 'dt_evento' as confirmed date field. This loses ~400 cases out of 350k.
+    
+    Using residential municipality ('municipio_res') as case location, as its the only location field other than 
+    'bairro' (neighborhood), which is missing in ~2/5 cases and cannot be easily geocoded with mapbox.
+    
+    
+    """
+    with open(raw_data_file, "r") as f:
+        reader = csv.DictReader(f,delimiter=';')
```

Comment:
nit, not worth holding up review: might want a space after the comma in the DictReader arguments for consistency.